include/sharg/detail/format_help.hpp

```diff
@@ -491,28 +490,28 @@ LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
 OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH
 DAMAGE.)"};
 
-        stream << std::string(80, '=') << "\n"
-               << in_bold("Copyright information for " + meta.app_name + ":\n")
-               << std::string(80, '-') << '\n';
+        std::cout << std::string(80, '=') << "\n"
```

Comment:
```suggestion
        std::cout << std::string(80, '=') << '\n'
```

Can you do a search/replace for `"\n"` to `'\n'`?
If there are many results, we can do a PR for this.

Reply:
There are quite a few :see_no_evil: I will change them in a separate PR